src/js/hooks/cycleCount/useCountStepTable.jsx

```diff
@@ -140,13 +140,21 @@ const useCountStepTable = ({
       const isEdited = initialValue !== value;
       // When the input is blurred, we'll call the table meta's updateData function
       const onBlur = () => {
-        if (isEdited) {
-          table.options.meta?.updateData(cycleCountId, original.id, id, value);
+        if (!isEdited) return;
+
+        const parsedValue = parseInt(value, 10) || 0;
```

Comment:
I think you could move it into the if statement, because you are not using this later

Reply:
It should be moved to that if. What happens when you try to use `parseInt` on an object? Won't it throw an exception?